frontend/src/components/Alerts/FailedRequestAlert.tsx

```diff
@@ -28,7 +28,7 @@ export const FailedRequestAlertContent = ({ message }: { message: string }) => {
             </StyledAlertTitle>
             <Indent>
                 <Button as={Typography} variant="ghost" color="secondary">
-                    {TranslateText(message)}
+                    {translatedMessage}
```

Comment:
This is very picky but for consistency I would rather have 

`<FailedRequestAlertContent message={'Failed to rerun mission'} />`
```
<Button as={Typography} variant="ghost" color="secondary">
                    {TranslateText(message)}
```

Reply:
The issue is that the message cannot always be translated as a full string, instead part of it need to be translated separately. The message for instance can contain a custom message from the backend. We may also want to include robot names, etc.